applications/CompressiblePotentialFlowApplication/custom_utilities/potential_flow_utilities.cpp

```diff
@@ -984,6 +984,36 @@ double CalculateArea(TContainerType& rContainer)
     return area;
 }
 
+template <int Dim, int NumNodes>
+void ComputePotentialJump(ModelPart& rWakeModelPart)
+{
+    const array_1d<double, 3>& vinfinity = rWakeModelPart.GetProcessInfo()[FREE_STREAM_VELOCITY];
+    const double vinfinity_norm = sqrt(inner_prod(vinfinity, vinfinity));
+
+    for (auto& r_elem : rWakeModelPart.Elements()) {
```

Comment:
Since this is anyway computed at the nodes, shall we just loop over the nodes?

Reply:
Currently the `wake_sub_model_part` only has Elements. I thought one could loop the nodes if only the elements are added but seems we can't (the nodes are defined in the root model part and wake_sub_model_part has no nodes). We could fill this with nodes in define_wake_process.cpp, but that requires changes there as well. Do you think it is worth doing that just for the potential jump?

Reply:
Ok, I thought that we added the nodes too. Then I agree it is not worth it.